test_sets/External_Consent_Status_OFF.txt

```diff
@@ -0,0 +1,31 @@
+./test_scripts/Polices/External_UCS/External_Consent_OFF/004_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_off_data_consent_disallowed.lua Check_that_SDL_returns_DISALLOWED_when_disallowed_by_external_consent_entities_off_exists_externalConsentStatus_OFF_and_data_consent_disallowed
+./test_scripts/Polices/External_UCS/External_Consent_OFF/005_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_off_user_consent_allowed.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_off_exists_externalConsentStatus_OFF_and_functional_group_allowed_data_consent_allowed
+./test_scripts/Polices/External_UCS/External_Consent_OFF/006_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_off_user_consent_disallowed.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_off_exists_externalConsentStatus_OFF_and_functional_group_disallowed_by_user
+./test_scripts/Polices/External_UCS/External_Consent_OFF/007_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_omitted_data_consent_disallowed.lua Check_that_SDL_returns_DISALLOWED_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_OFF_and_data_consent_disallowed
+./test_scripts/Polices/External_UCS/External_Consent_OFF/008_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_omitted_user_consent_allowed.lua Check_that_SDL_returns_SUCCESS_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_OFF_and_functional_group_allowed_data_consent_allowed
+./test_scripts/Polices/External_UCS/External_Consent_OFF/009_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_omitted_user_consent_disallowed.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_OFF_and_functional_group_disallowed_by_user
+./test_scripts/Polices/External_UCS/External_Consent_OFF/010_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_omitted_user_consent_omitted.lua Check_that_SDL_returns_USER_DISALLOWED_when_disallowed_by_external_consent_entities_on/off_omitted_externalConsentStatus_OFF_and_functional_group_omitted
+./test_scripts/Polices/External_UCS/External_Consent_OFF/011_ATF_Policies_External_Consent_OFF_through_ignition_cycles_allowed_RPCs.lua Check_that_SDL_uses_value_externalConsentStatus_OFF_when_disallowed_by_external_consent_entities_on_exists_allowed_RPCs_during_ign_cycles
+./test_scripts/Polices/External_UCS/External_Consent_OFF/012_ATF_Policies_External_Consent_OFF_through_ignition_cycles_user_disallowed_RPCs.lua Check_that_SDL_uses_value_externalConsentStatus_OFF_when_disallowed_by_external_consent_entities_off_exists_during_ign_cycles
```

Comment:
@istoimenova, I see existing "ATF_Policies_External_Consent_OFF_through_ignition_cycles.lua" script is split into 011 - 013 ones 
I'm Ok with 013 script, but please clarify why do we need 011 and 012.
Are these cases not covered in original script?